docs/UserGuide.md

```diff
@@ -152,12 +152,12 @@ Adds a user-defined anime into the anime list.
 Format: `add n/NAME [e/EPISODE] [s/STATUS] [g/GENRE] [g/GENRE] ...`
 
 Notes:
-* If multiple `n/NAME` are given in the command, the last provided `NAME` will be used
-* If multiple `e/EPISODE` are given in the command, the last provided `EPISODE` will be used
-* If multiple `s/STATUS` are given in the command, the last provided `STATUS` will be used
+* If more than one `n/NAME` is given in the command, the last provided `NAME` will be used
+* If more than one `e/EPISODE` is given in the command, the last provided `EPISODE` will be used
+* If more than one `s/STATUS` is given in the command, the last provided `STATUS` will be used
 * If `EPISODE` is not provided, it will be set to `0`
 * If `STATUS` is not provided, it will be set to `watching`
-* All genres provided must be valid for the command to be successful
+* If genres are provided, all of them must be valid for the command to be successful
```

Comment:
makes this sentence very wordy

Reply:
i want to avoid sounding like genres must be provided